apps/explorer/src/pages/transaction-result/programmable-transaction-view/index.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { type ProgrammableTransaction } from '@mysten/sui.js';
+
+import { Commands } from './Commands';
+import { Inputs } from './Inputs';
+
+interface Props {
+    transaction: ProgrammableTransaction;
+}
+
+export function ProgrammableTransactionView({ transaction }: Props) {
+    return (
+        <>
+            <div
+                data-testid="programmable-transactions-inputs"
+                className="mt-8"
+            >
+                <section className="pt-4 pb-4">
+                    <Inputs inputs={transaction.inputs} />
+                </section>
+            </div>
+            <div
+                data-testid="programmable-transactions-commands"
+                className="mt-8"
+            >
+                <section className="pt-4 pb-4">
```

Comment:
I wanted to use margin so that its the same as the `Gas & Storage Fees` but wasnt able to extend margin bottom as seen in gif here. Will update the section to paddings.
![margins](https://user-images.githubusercontent.com/127577476/226681624-dd2df064-38d8-4da0-9ecf-0ec18f46b1a3.gif)